testing-modules/junit-5/pom.xml

```diff
@@ -135,7 +135,6 @@
         <log4j2.version>2.17.1</log4j2.version>
         <powermock.version>2.0.9</powermock.version>
         <spring.version>5.0.1.RELEASE</spring.version>
-        <mockito.version>3.3.0</mockito.version>
```

Comment:
running mvn clean install fails for this module

Reply:
Rollbacked